apps/minifront/src/state/send/index.ts

```diff
@@ -36,6 +36,8 @@ export interface SendSlice {
   setFeeTier: (feeTier: FeeTier_Tier) => void;
   sendTx: () => Promise<void>;
   txInProgress: boolean;
+  isSendingMax: boolean;
+  SetIsSendingMax: (isSendingMax: boolean) => void;
```

Comment:
```suggestion
  setIsSendingMax: (isSendingMax: boolean) => void;
```

(and update throughout codebase)

Reply:
changed naming convention